spec/might/filter_parameters_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'might/filter_parameters'
+require 'might/filter_parameter_definition'
+require 'might/filter_parameter'
+
+RSpec.describe Might::FilterParameters do
+  let(:parameter_definition) { Might::FilterParameterDefinition.new(:height) }
+  let(:parameter) { Might::FilterParameter.new('146', 'eq', parameter_definition) }
+  let(:filter_parameters) { described_class.new([parameter]) }
+
+  describe '#[]' do
+    subject { filter_parameters[parameter_name] }
+
+    context 'when existing parameter name given' do
+      let(:parameter_name) { 'height' }
+
+      it 'returns parameter with this name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.